.github/ISSUE_TEMPLATE/release-team-lead-progress.md

```diff
@@ -0,0 +1,241 @@
+- [ ] Update the SIG Release groups in the `k/k8s.io/groups/sig-release/groups.yaml` with the following:
+  - [ ] Add Lead and Lead shadows to members of `k8s-infra-release-viewers`
+  - [ ] Add EA, Lead, Lead shadows, comms lead, comms shadows to members of `release-comms`
+  - [ ] Add EA, Lead and Lead shadows to members of `release-managers`
+  - [ ] Add EA, Lead and Lead shadows to managers of `release-team`
+  - [ ] Add Role Leads and Role shadows to members of `release-team`
+  - [ ] Add Role shadows and Lead shadows to members of `release-team-shadows`
+  - [ ] Add EA to manager of release-team-shadows, if EA is owner of `release-team-shadows` already then add Lead to manager of `release-team-shadows`
+  - [ ] Add Bug Triage Lead and shadows to `k8s-infra-rbac-triageparty-release`
+- [ ] Ensured that there is a branch manager available for cutting x.y.0-alpha.1
+
+- Assist the Enhancements Lead in collecting planned work from SIGs
+- [ ] Discussed and scheduled a weekly Release Team meetings on a day that is most acceptable to the team. Invite the `kubernetes-sig-release` group.
+- [ ] Poll Release Team membership and schedule a weekly alternate meeting to better enable more attendance outside of the Americas.
+- [ ] Major release cycle events have been added to the Kubernetes Release Calendar with one week in advance reminders set. (defer to the [handbook for more information](https://github.com/kubernetes/sig-release/tree/master/release-team/role-handbooks/release-team-lead#working-with-the-release-team-calendar))
+- [ ] Checked in with ci-signal and branch managers if x.y.0-alpha.2 is ok to be released and [master-blocking](https://testgrid.k8s.io/sig-release-master-blocking) tests are all green
+
+
+**A week before Enhancements Freeze:**
+- [ ] Remind the community about Enhancements Freeze reminder send out. Example email: [1.26](https://groups.google.com/a/kubernetes.io/g/dev/c/pJZC2cnkeJ8); It may also be useful to send a short version to the `#sig-release` and `#chairs-and-techleads` Slack channels referencing to the k-dev email.
+
+### 3. Enhancements freeze up to Release Halfway Point (~Week 5 - 7)
+
+**General Tasks:**
+- Bring exceptions to the #sig-release Slack channel and to Release Team meetings, and make sure SIG representatives for the exception(s) know to attend and discuss if necessary.
+- Begin casual observation of issues, CI signal, test flakes, and critical PRs
+- [ ] Release Team Retro is scheduled shortly after the "Release Halfway Point" and a host is selected
+
+**Release Cut Alpha 3:**
+- [ ] Checked in with ci-signal and branch managers if 1.XX.0-alpha.3 is ok to be released and [master-blocking](https://testgrid.k8s.io/sig-release-master-blocking) tests are all green
+- [ ] Checked in with Release Notes team on release notes progress after the release notes for 1.XX.0-alpha.3 is generated
```

Comment:
I see many `release notes` words in this sentence. I think the last one should be changed to `tag`. As Release Notes Team we should generate the release notes after the tag/release for `1.XX.0-alpha.3` is generated/published.

```suggestion
- [ ] Checked in with Release Notes team on release notes progress after the tag for 1.XX.0-alpha.3 is generated
```